ravenframework/CustomDrivers/DriverUtils.py

```diff
@@ -124,7 +125,11 @@ def checkVersions():
     sys.path.append(scriptDir)
   else:
     remove = False
-  import library_handler as LH
+  try:
+    import library_handler as LH
```

Comment:
importing library_handler fails if installed from pip, but not sure if a try except is the best way to handle this.

Reply:
Which means we will skip libraries check if use pip for installation, right?

Reply:
Yes, this will skip library checks in the pip installation.